nncf/quantization/quantize_functions.py

```diff
@@ -147,9 +147,8 @@ def backward(ctx, grad_output):
 
 
 def get_scale_zp_from_input_low_input_high(level_low, level_high, input_low, input_high):
-    levels = level_high - level_low + 1
+    levels = level_high - level_low
```

Comment:
Is this correct? Now `levels` does not represent the entire count of levels, which is not good.

Reply:
It is mistake. I return back the formula